src/sentry/options/defaults.py

```diff
@@ -1793,6 +1793,10 @@
     "hybrid_cloud.disable_relative_upload_urls", default=False, flags=FLAG_AUTOMATOR_MODIFIABLE
 )
 
+# Flagpole Rollout
+register("features", default={}, flags=FLAG_AUTOMATOR_MODIFIABLE)
+register("hybrid-cloud.flagpole_rollout_phase", default=0, flags=FLAG_AUTOMATOR_MODIFIABLE)
```

Comment:
```suggestion
register("hybridcloud.flagpole_rollout_phase", default=0, flags=FLAG_AUTOMATOR_MODIFIABLE)
```

Could we normalize on `hybridcloud` or `hybrid_cloud`?

Reply:
I'm not too fussy on which one we use as we have two already :cry:

Reply:
Should be fixed now